etc/schema.graphqls

```diff
@@ -30,13 +30,27 @@ type AdyenPaymentMethods {
 
 type AdyenPaymentMethodsResponse {
     paymentMethods: [AdyenPaymentMethodsArray]
+    storedPaymentMethods: [AdyenStoredPaymentMethodsArray]
 }
 
 type Order {
     cart_id: String! @doc(description: "Cart ID.")
     adyen_payment_status: AdyenPaymentStatus @resolver(class: "Adyen\\Payment\\Model\\Resolver\\GetAdyenPaymentStatus")
 }
 
+type AdyenStoredPaymentMethodsArray {
```

Comment:
Hi @larsroettig,

First off, thanks for opening this PR. Some very minor notes. Could you update the descriptions of the fields according to the parameter descriptions on [this page](https://docs.adyen.com/api-explorer/#/CheckoutService/v68/post/paymentMethods__resParam_storedPaymentMethods)?

Reply:
@Morerice I updated the PR pls review agian added also iban and onwer name to support 100 % possible requirements